src/symspell.rs

```diff
@@ -762,4 +859,25 @@ mod tests {
         assert_eq!(10, results[0].distance);
         assert_eq!(0, results[0].count);
     }
+
+    #[test]
+    fn test_lookup_compound_with_bigrams() {
+        let edit_distance_max = 2;
+        let mut sym_spell = SymSpell::<UnicodeStringStrategy>::default();
+        sym_spell.load_dictionary("./data/frequency_dictionary_en_82_765.txt", 0, 1, " ");
+        sym_spell.load_bigram_dictionary(
+            "./data/frequency_bigramdictionary_en_243_342.txt",
```

Comment:
First of all, i think we should include the bigram dictionary in the repo.
Now the test is not hitting the bigram branch because the file is missing.

But also including the dictionary from `symspellpy` port it fallbacks on Naive Bayes branch.
I don't know why yet, but I dig into the problem

Reply:
Added it, got left out because of .gitignore.

Reply:
>But also including the dictionary from symspellpy port it fallbacks on Naive Bayes branch.

Any pointers/link to this? The current code already fallback on the naive bayes logic.

Reply:
> Any pointers/link to this? The current code already fallback on the naive bayes logic.
i've taken this [dictionary](https://github.com/mammothb/symspellpy/blob/54662d219ae73030bf715b494e57a03789851631/symspellpy/frequency_bigramdictionary_en_243_342.txt) from the python port and i launched the test as follow 
 `cargo tarpaulin -o Html -- test_lookup_compound_with_bigrams `
From the report it seems that line 512 
`let count2: i64 = match self.bigrams.get(&suggestion_split.term)` 
return always `None` for some reason, maybe it could be some mismatching in formatting between `self.bigrams` HashMap and `suggestion_split.term`, but i don't know.

Reply:
Ok, i think the error lies in how  `bigram` hasmap is built.
Currently it's formatted as `{}{}` but is confronted with `suggestion_split.term` which is formatted as `{} {}`.
This is not visible with the current sentence because the splits never match a bigram entry, but using for example `Can yu readthis` -> `can you read this`